src/forms/select/Select.tsx

```diff
@@ -28,6 +28,19 @@ export const SelectOption: React.FunctionComponent<SelectOption> = ({value, sele
     );
 };
 
+type SelectOptionGroup = {
+    label: string;
+    disabled?: boolean;
+};
+
+export const SelectOptionGroup: React.FunctionComponent<SelectOptionGroup> = ({label, disabled, children}) => {
+    return (
+        <optgroup label={label} disabled={disabled}>
+            {children}
+        </optgroup>
```

Comment:
if disable is true here. Will it disable child options as well ?
Can u add story for disabled group option ?

Reply:
Yes It will disabled child options as well. Added the story as well.
![disabled-option-group](https://user-images.githubusercontent.com/70933150/120293869-4a9b7980-c2e3-11eb-84b1-d602a7732110.gif)